src/iosrv/server_iv.c

```diff
@@ -832,6 +810,10 @@ void
 ds_iv_init()
 {
 	D_INIT_LIST_HEAD(&ds_iv_ns_list);
+	/* Let's set the topo to 32 to avoid cart IV failover,
+	 * which is not supported yet. XXX
+	 */
+	ds_iv_ns_tree_topo = crt_tree_topo(CRT_TREE_KNOMIAL, 32);
```

Comment:
Is there a ticket to circle back on this when it is supported?

Reply:
I don't know. The workaround exists in master already; this PR just moves it here.